app/notifications/validators.py

```diff
@@ -159,7 +157,8 @@ def validate_and_return_extended_phone_number_info(service, send_to, key_type, c
 
         recipient_data = _get_extended_phone_number_info(phone_number, send_to)
 
-        if recipient_data["international"] and check_intl_sms_limit:
+        # UK_PREFIX includes standard domestic numbers, and crown dependency
+        if (not phone_number.is_uk_phone_number()) != UK_PREFIX and check_intl_sms_limit:
```

Comment:
`!= UK_PREFIX` is wrong 

Swap the order and no brackets are needed:
```suggestion
        if check_intl_sms_limit and not phone_number.is_uk_phone_number():
```